packages/unmock-core/src/interfaces.ts

```diff
@@ -73,6 +73,19 @@ export interface IMock {
   response: ISerializedResponse;
 }
 
+export interface IResponseCreatorFactoryInput {
+  mockGenerator: () => IMock[];
```

Comment:
Is the mock generator used for anything anymore? Now I'm quite confused of all the curried definitions :)

Reply:
Ah, just in your original part of the code in `responseCreatorFactory`, where `opts: IResponseCreatorFactoryInput` is passed, to pass `opts.mockGenerator` to `httpRequestMatcherFactory`.